<ui version="4.0" >
 <class>Client</class>
 <widget class="QMainWindow" name="Client" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection" >
    <property name="title" >
     <string>Actions</string>
    </property>
    <addaction name="actionSignUp" />
    <addaction name="actionSignIn" />
    <addaction name="actionSignOut" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>About</string>
    </property>
    <addaction name="actionInformation" />
   </widget>
   <addaction name="menuConnection" />
   <addaction name="menuAbout" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBarAction" >
   <property name="windowTitle" >
    <string>Action</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionSignUp" />
   <addaction name="actionSignIn" />
   <addaction name="actionSignOut" />
   <addaction name="separator" />
   <addaction name="actionQuit" />
  </widget>
  <widget class="QToolBar" name="toolBarAbout" >
   <property name="windowTitle" >
    <string>About</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionInformation" />
  </widget>
  <action name="actionSignIn" >
   <property name="icon" >
    <iconset>images/connect.png</iconset>
   </property>
   <property name="text" >
    <string>Sign in</string>
   </property>
  </action>
  <action name="actionSignOut" >
   <property name="icon" >
    <iconset>images/disconnect.png</iconset>
   </property>
   <property name="text" >
    <string>Sign out</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset>images/door.png</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSignUp" >
   <property name="icon" >
    <iconset>images/database_save.png</iconset>
   </property>
   <property name="text" >
    <string>Sign up</string>
   </property>
  </action>
  <action name="actionInformation" >
   <property name="icon" >
    <iconset>images/information.png</iconset>
   </property>
   <property name="text" >
    <string>Information</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
